TimberAPIExample/Examples/GameObjectModifier/EntityInstantiatorConfigurator.cs

```diff
@@ -0,0 +1,15 @@
+using Bindito.Core;
+using TimberAPIExample.AutoConfiguratorInstaller;
+using TimberbornAPI.EntityInstantiatorSystem;
+
+namespace TimberAPIExample.Examples.GameObjectModifier
+{
+    public class EntityInstantiatorConfigurator : IInGameConfigurator
```

Comment:
Don't forget to rename this as well, if we agree on something else than `EntityInstantiator`